src/pages/Contacts.tsx

```diff
@@ -51,33 +118,83 @@ const Contacts = () => {
-							/>
-							<input
-								type="text"
-								placeholder="Email"
-								className="px-8 py-2 rounded-md bg-inputBg text-inputCaption"
-							/>
-							<textarea
-								placeholder="Message"
-								className="px-8 py-2 rounded-md bg-inputBg text-inputCaption  h-[200px]"
-							/>
-							<button className="bg-custom-gradient text-neutral-white w-[100%] h-[44px] rounded-[10px]">
+					<div className="gap-6 mt-1 laptop:gap-5 laptop:w-[50%] w-full flex flex-col laptop:px-5 laptop:h-full justify-between">
+						<form
+							onSubmit={handleSubmit(onSubmit)}
+							className="flex flex-col gap-4 w-full h-auto"
+						>
+							<div className="relative">
+								<input
+									type="text"
+									placeholder="Last name"
+									{...register('lastName')}
+									className="px-8 py-2 rounded-md bg-inputBg text-inputCaption w-full"
+								/>
+								{errors.lastName && (
+									<p className="text-action-error text-[10px] absolute bottom-[-16px] left-0">
+										{errors.lastName?.message}
+									</p>
+								)}
+							</div>
+
+							<div className="relative">
+								<input
+									type="text"
+									placeholder="First name"
+									{...register('firstName')}
+									className="px-8 py-2 rounded-md bg-inputBg text-inputCaption w-full"
+								/>
+								{errors.firstName && (
+									<p className="text-action-error text-[10px] absolute bottom-[-16px] left-0">
+										{errors.firstName?.message}
+									</p>
+								)}
+							</div>
+
+							<div className="relative">
+								<input
+									type="text"
+									placeholder="Subject"
+									{...register('subject')}
+									className="px-8 py-2 rounded-md bg-inputBg text-inputCaption w-full"
+								/>
+								{errors.subject && (
+									<p className="text-action-error text-[10px] absolute bottom-[-16px] left-0">
+										{errors.subject?.message}
+									</p>
+								)}
+							</div>
+
+							<div className="relative">
+								<input
+									type="text"
+									placeholder="Email"
+									{...register('email')}
+									className="px-8 py-2 rounded-md bg-inputBg text-inputCaption w-full"
+								/>
+								{errors.email && (
+									<p className="text-action-error text-[10px] absolute bottom-[-16px] left-0">
+										{errors.email?.message}
+									</p>
+								)}
+							</div>
+
+							<div className="relative">
+								<textarea
+									rows={1000}
+									cols={1000}
+									placeholder="Message"
+									{...register('message')}
+									className="px-8 py-2 rounded-md bg-inputBg text-inputCaption h-[200px] w-full"
+								/>
+								{errors.message && (
+									<p className="text-action-error text-[10px] absolute bottom-[-16px] left-0">
+										{errors.message?.message}
+									</p>
+								)}
+							</div>
+
+							<button className="bg-custom-gradient text-neutral-white w-full h-[44px] rounded-[10px] mt-4">
```

Comment:
Also add type="submit" to this button, to insure the onSubmit work well.
Apart from that, either the code or the functionality are working well. Great job!